### PR TITLE
feat : added category

### PR DESCRIPTION
**Pull request by Sharmaryan**
- user can select category from landing page and then redirects to the product page

### CODE REVIEW

File: src/components/Category/Category.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import "./Category.css";
+import { Link } from "react-router-dom";
+import { useProduct } from "context/product-context";
+
+export const Category = () => {
+  const { productsDispatch } = useProduct();
+  return (
+    <div>
+      {" "}
```
Comment (ravikumar0403) [inline src/components/Category/Category.jsx]: remove this `{" "}`

File: src/components/Category/Category.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            className="category-image"
+            alt="car"
+          />
+          <p className="category-title">
+            <Link
+              to="/products"
+              className="text-decorations text-white"
+              onClick={() => productsDispatch({ type: "BUSINSESS" })}
+            >
+              {" "}
```
Comment (ravikumar0403) [inline src/components/Category/Category.jsx]: here as well

File: src/components/Sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ export const SideBar = () => {
     range,
     productsDispatch,
   } = useProduct();
-
+console.log(business)
```
Comment (ravikumar0403) [inline src/components/Sidebar/Sidebar.jsx]: you missed removing this console log

**Review by shashikantmaurya0007 — APPROVED** [review]
Code is clean . beautiful.you can go and merge

**Review by ravikumar0403 — APPROVED** [review]
Looks good, i've left a few  comments

File: src/components/Category/Category.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+              Business
+            </Link>
+          </p>
+        </div>
+        <div className="category">
+          <img
+            src="https://images-na.ssl-images-amazon.com/images/I/41+CqNWoutS._SX460_BO1,204,203,200_.jpg"
```
Comment (Chiragmodi01) [inline src/components/Category/Category.jsx:31]: you can put all these images links in a separate file and import here. jsx will look cleaner.

File: src/components/Sidebar/Sidebar.jsx (hunk 3 above)
Comment (Chiragmodi01) [inline src/components/Sidebar/Sidebar.jsx]: remove logs before pushing

**Review by Chiragmodi01 — COMMENTED** [review]
- Filters section doesn't have proper height in the preview.
- In `slider-container` , the text will look good in the color white also give text some padding on left.

Overall code structure and UI looks good, just added few sanity comments.

**Pull request merged**